x-pack/plugin/esql/qa/testFixtures/src/main/resources/spatial.csv-spec

```diff
@@ -49,22 +59,32 @@ ZAH            | POINT(60.9007085300982 29.47529417462647)    | Zahedan Int'l
 ;
 
 convertCartesianFromLong#[skip:-8.11.99, reason:spatial type cartesian_point only added in 8.12]
+// tag::to_cartesianpoint-long[]
 row long = 5009771769843126025
-| eval pt = to_cartesianpoint(long);
+| eval pt = to_cartesianpoint(long)
```

Comment:
It'd be so nice if we could hex encode here. Oh well.

Reply:
A hex literal? That sounds nice! It is a language feature though, so not in scope for this PR.